packages/contracts/scripts/deploy/dollar/solidityScripting/00_Constants.s.sol

```diff
@@ -55,42 +52,42 @@ contract Constants is Script {
     ];
 
     uint256[] public lpAmounts = [
-        1301000000000000000,
-        3500000000000000000000,
+        1301e15,
+        35e20,
         9351040526163838324896,
         44739174270101943975392,
         74603879373206500005186,
-        2483850000000000000000,
+        248385e16,
         1878674425540571814543,
         8991650309086743220575,
         1111050988607803612915,
         4459109737462155546375,
-        21723000000000000000000,
-        38555895255762442000000,
+        21723e18,
+        38555895255762442e6,
         5919236274824521937931,
         1569191092350025897388,
         10201450658519659933880,
         890339946944155414434,
         5021119790948940093253,
-        761000000000000000000,
+        761e18,
         49172294677407855270013,
         25055256356185888278372,
         1576757078627228869179,
-        3664000000000000000000,
+        3664e18,
         1902189597146391302863,
         34959771702943278635904,
         9380006436252701023610,
         6266995559166564365470,
-        100000000000000000000,
+        1e20,
         3696476262155265118082,
-        740480000000000000000,
-        2266000000000000000000,
+        74048e16,
+        2266e18,
         1480607760433248019987,
         24702171480214199310951,
-        605000000000000000000,
+        605e18,
         1694766661387270251234,
-        14857000000000000000000,
-        26000000000000000000
+        14857e18,
+        26e18
     ];
```

Comment:
This is nice and succinct but I presume that real money is on the line here and we should be especially careful to validate that these were all converted correctly.

Reply:
> This is nice and succinct but I presume that real money is on the line here and we should be especially careful to validate that these were all converted correctly.

yes for sure